### PR TITLE
Update structures.js

### PR DESCRIPTION
**Pull request by Aumptl02**
Thoda change kiya hai, ek baar check kar lena.

### CODE REVIEW

File: structures.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 const user = {
+  id: Number; // We need id too, to link with the user for easy working on our side. It will not be visible to the user.
```
Comment (adnan007d) [inline structures.js:2]: This will be a string as we will be using firestore's inbuilt id

File: structures.js [hunk 2]
```diff
@@ -15,6 +16,7 @@ const post = {
   description: String,
   user: user,
   image: String, // Not mandatory and this will only store the url of the stored image
+  userProfileImage: String,
```
Comment (adnan007d) [inline structures.js:19]: No need of this as the user object will conatain the profileImage

**Review by adnan007d — CHANGES_REQUESTED** [review]

File: structures.js (hunk 1 above)
Comment (Aumptl02) [reply structures.js:2]: Okay

File: structures.js (hunk 2 above)
Comment (Aumptl02) [reply structures.js:19]: Oh, Got it..

**Review by adnan007d — APPROVED** [review]

**Pull request merged**